src/cmd/forwarder-agent/app/forwarder_agent_test.go

```diff
@@ -375,6 +375,7 @@ var _ = Describe("App", func() {
 				<-otelTraceServer.requests
 			}
 			// test-title events
+			Eventually(otelLogsServer.requests).Should(HaveLen(1))
```

Comment:
Seems like this change is breaking some tests further down. `otelLogsServer` will not receive anything when log forwarding to OTel Collectors is disabled.